app/src/main/java/com/name/name/musicappmvp/ui/main/MainActivity.kt

```diff
@@ -0,0 +1,160 @@
+package com.name.name.musicappmvp.ui.main
+
+import android.Manifest
+import android.content.ComponentName
+import android.content.Context
+import android.content.Intent
+import android.content.ServiceConnection
+import android.content.pm.PackageManager
+import android.os.Build
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.IBinder
+import android.text.TextUtils
+import android.view.View
+import android.widget.SeekBar
+import android.widget.SeekBar.*
+import com.name.name.musicappmvp.R
+import com.name.name.musicappmvp.data.model.LocalSong
+import com.name.name.musicappmvp.data.repository.SongRepository
+import com.name.name.musicappmvp.service.PlaySongService
+import com.name.name.musicappmvp.ui.adapter.LocalMusicAdapter
+import kotlinx.android.synthetic.main.activity_main.*
+import java.util.*
+import kotlin.concurrent.thread
+
+class MainActivity : AppCompatActivity(), View.OnClickListener, LocalMusicInterface.View {
+    private var localSongs = mutableListOf<LocalSong>()
+    private var isBind = false
+    private var localMusicPresenter: LocalMusicPresenter? = null
```

Comment:
class giao tiếp với nhau qua Interface
```suggestion
    private var localMusicPresenter: MainContract.Presenter ? = null
```